web/src/pages/Subscribe.jsx

```diff
@@ -0,0 +1,46 @@
+import { Flex, Typography } from "antd";
+
+import { SubscriptionLogo } from "../components/SubscriptionLogo";
+
+function Subscribe() {
+	return (
+		<Flex
+			align="center"
+			justify="center"
+			style={{ height: "100vh", padding: "0 40px" }}
+		>
+			<Flex
+				vertical
+				align="center"
+				justify="center"
+				gap={44}
+				style={{ maxWidth: "662px" }}
+			>
+				<div style={{ width: "260px", height: "183px", overflow: "hidden" }}>
+					<SubscriptionLogo />
+				</div>
+
+				<Flex vertical gap={24} align="center">
+					<Typography.Title level={2} align="center">
```

Comment:
Recheck Ant Design's Typography, please. The title does not have an align or justify prop, but it supports inline text alignment styles.

Reply:
Sorry for late response.

After checking the Antd documentation and TypeScript definitions, I found that `Typography.Title` does not accept align or justify props. While the alignment appeared to work due to CSS inheritance, it's better to handle text alignment through proper CSS methods or parent containers to maintain type safety and prevent potential issues. Thank you @ebrahimbeiati for catching this.